lib/bootstrap/bundler.rb

```diff
@@ -48,8 +48,27 @@ def self.reset_paths!
       # repository. This basically enabled the offline feature to work as
       # we remove the gems from the vendor directory before packaging.
       ::Bundler::Source::Rubygems.module_exec do
```

Comment:
I'm having trouble understanding this patch. I wonder if it was a workaround for an upstream bug that has been fixed now? https://github.com/rubygems/rubygems/commit/b9a2d4d5395f4602a7073aa7ce63ee7f2dd6f404 

Experimenting with just removing the patch all together.